spec/support/authentication_helper.rb

```diff
@@ -1,10 +1,10 @@
 module AuthenticationHelper
-	def sign_in_as(user)
-		visit new_user_session_path
+  def sign_in_as(user)
+    visit new_user_session_path
 
-		fill_in 'Email', with: user.email
-		fill_in 'Password', with: user.password
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: user.password
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.